src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -86,21 +85,6 @@ export const DetailsView: React.FunctionComponent<DetailsViewProps> = ({
     }
   }, [queryStatus, queryIsFetching, hasNextPage, fetchNextPage, inView])
 
-  const tableRef = useRef(null)
-
-  useEffect(() => {
```

Comment:
column-resizer is not compatible with functional components because it requires class component lifecycle methods to work